src/server/pfs/s3/bucket.go

```diff
@@ -192,12 +190,13 @@ func (c *controller) CreateBucket(r *http.Request, bucketName string) error {
 }
 
 func (c *controller) DeleteBucket(r *http.Request, bucketName string) error {
+	c.logger.Debugf("DeleteBucket: %+v", bucketName)
```

Comment:
Would it make sense to use Pachyderm's internal log library to log GRPC fn call names more automatically?https://github.com/pachyderm/pachyderm/blob/b94abf8d1cbaccd8539c5bac85fdd90d43a6464e/src/server/pkg/log/log.go#L89

Example: https://github.com/pachyderm/pachyderm/blob/master/src/server/pps/server/api_server.go#L887-L889

Reply:
The `Log` function is specific to gRPC calls, and these aren't. I could make them log something syntactically more similar to gRPC log statements though, if you think that would be better?

Reply:
Let me know if you want to address this at some point and I can open a separate ticket to fix.